### ISSUE TITLE
Modal: Help and close icon button not reachable via keyboard (2.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#852]
### Expected behavior

A keyboard can be used to tab to the buttons and they can be "clicked" using keyboard key. They have the correct semantic role - button. 
### Actual behavior

The buttons cannot be reached when tabbing through the modal. Thus help cannot be opened.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.9.0](https://github.com/blackbaud/skyux/releases/tag/v1.9.0)
